### PR TITLE
[WIP] DevKit updates

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
This is the first try of an dev-kit created PR.

* [x] fix packagist name for seo bundle
* [x] be aware of project configuration when creating travis matrix (it is currently coppied from the repo as plain text)
* [x] fix error `[ERROR] Failed with message: Validation Failed: Field "head" is invalid, for  resource "PullRequest"`
* [ ] discuss about the buttons in README.md
* [x] configuration to controll `Makefile` tasks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
-# Symfony CMF Seo Bundle
+# Symfony Cmf Seo Bundle
```
Comment (ElectricMaxxx) [inline README.md]: this change comes from packagist

File: README.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+[![Total Downloads](https://poser.pugx.org/symfony-cmf/seo-bundle/downloads)](https://packagist.org/packages/symfony-cmf/seo-bundle)
+[![Monthly Downloads](https://poser.pugx.org/symfony-cmf/seo-bundle/d/monthly)](https://packagist.org/packages/symfony-cmf/seo-bundle)
+[![Daily Downloads](https://poser.pugx.org/symfony-cmf/seo-bundle/d/daily)](https://packagist.org/packages/symfony-cmf/seo-bundle)
 
-It provides a solution to make content bundles
-aware for Search Engine Optimisation (SEO).
+Branch | Travis | Coveralls |
+------ | ------ | --------- |
+2.0   | [![Build Status][travis_legacy_badge]][travis_legacy_link]     | [![Coverage Status][coveralls_legacy_badge]][coveralls_legacy_link]     |
+2.0   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     |
```
Comment (dbu) [inline README.md]: 2.0 is duplicated

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+### Environment
+
+#### Symfony packages
+
+```
+$ composer show --latest 'symfony-cmf/*'
+```
+
+#### Symfony packages
```
Comment (dbu) [inline .github/ISSUE_TEMPLATE.md]: this is duplicated

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+```
+$ composer show --latest 'symfony/*'
+```
+
+#### PHP version
```
Comment (dbu) [inline .github/ISSUE_TEMPLATE.md]: lets remove this section, this will very rarely be relevant

**Comment by ElectricMaxxx** [comment]
I currently have a fork of dev-kit running localy, will push the code on a branch later to compare the result with the code, which generates it.

File: README.md (hunk 2 above)
Comment (ElectricMaxxx) [reply README.md]: Should depend on project config also.

Comment (ElectricMaxxx) [reply README.md]: mh. .. it comes from parameter passed to the file, i will look.

File: README.md [hunk 5]
```diff
@@ -1,15 +1,20 @@
...
+master | [![Build Status][travis_unstable_badge]][travis_unstable_link] | [![Coverage Status][coveralls_unstable_badge]][coveralls_unstable_link] |
+
+Symfony CMF Search Engine Optimization Bundle
 
 ## Requirements
```
Comment (ElectricMaxxx) [inline README.md]: The requirements should be filled in from project configuration also. Doing so we got the same versions in readme as testing in travis.

Comment (ElectricMaxxx) [reply README.md]: done.

**Comment by ElectricMaxxx** [comment]
I silently added codecoverage tests through this PR, any opinions against it?

Seems to be nice from this state. Will have to fix master then by using testing component with its scripts.

**Comment by ElectricMaxxx** [comment]
I think here is an error in the destination version, we should target 2.0 and merge into master then, right? or shall we build different PR on each version?

**Comment by dbu** [comment]
i would only introduce this on the master branch. afaik with the cmf we do not aim to provide long term support of older versions, because it would be too much effort.

**Comment by ElectricMaxxx** [comment]
@dbu sure, i think the current config depends on the default branch situation of sonata. I will start with master and we could possibly add 2.1 then if there is a change after we released it.

**Comment by ElectricMaxxx** [comment]
closed in favor of #340

**Pull request closed without merging**